lesson08/script.js

```diff
@@ -24,8 +24,7 @@ function createList(value){
     document.getElementById('js-list').appendChild(fragment);
 }
 
-function getList(){
-    createLoading();
+function SetDataToLists(){
     return new Promise((resolve,reject) => {
     const lists = [
         {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
```

Comment:
I changed the function name because the process does not match the function name "getList ()".
The "CreateLoading ();" function shouldn't be included in this function, so I put out it.